test.js

```diff
@@ -109,9 +109,10 @@ test('should serialize nested errors', t => {
 	t.is(serialized.innerError.message, 'inner error');
 });
 
-test('should convert Date to ISO string', t => {
+test('should convert properties of type Date to ISO string', t => {
 	const error = new Error('foo');
-	error.birthdate = new Date(0);
+	const date = new Date(0)
+	error.date = date;
 	const serialized = serializeError(error);
-	t.deepEqual(serialized.birthdate, new Date(0));
+	t.is(serialized.date, date.toISOString());
 });
```

Comment:
Your test here was faulty. It's no longer passing after I fixed it.

Reply:
But that is not correct. The properties should be serialized. Using `JSON.stringify` is just an example in the readme. It's not the only use-case.

Reply:
> The properties should be serialized.

I did not know that, because:
```javascript
var er = new Error("foobar");
er.info = [1, 2, 3];
er.code = Symbol(9);
serializeError(er).info instanceof Array // => true 
typeof serializeError(er).code // => Symbol
```
So Arrays and Symbols right now are serialized, or not? I mean they retain their original type, why not Date?

Reply:
Array is a supported type in JSON, so it would naturally be preserved, but the `.code` case is indeed a bug.

Reply:
Hmm I think we hit a dead end here, or I don't understand it.
On the one hand you say
>  Using JSON.stringify is just an example in the readme. It's not the only use-case.

but on the other you justify the decision what types get converted on the JSON format:
>Array is a supported type in JSON, so it would naturally be preserved

So I have no clear guideline what the problem is `serialize-error` addresses - apart from circular dependencies of course - in order to contribute to the project.